tests/syft/grid/duet_test.py

```diff
@@ -43,36 +37,26 @@ def test_duet_searchable_functionality() -> None:
     del xp
     del xp2
 
-    duet.stop()
-
 
 @pytest.mark.slow
-def test_duet_exception_catching() -> None:
-    try:
-        duet = sy.Duet(host="127.0.0.1", port=5001)
-
-        xp = th.tensor([1, 2, 3]).tag("some", "diabetes", "data").send(duet)
-        xp2 = (
-            th.tensor([1, 2, 3])
-            .tag("some", "diabetes", "data")
-            .send(duet, searchable=True)
-        )
+def test_duet_exception_catching(duet: sy.Duet) -> None:
+    xp = th.tensor([1, 2, 3]).tag("some", "diabetes", "data").send(duet)
+    xp2 = (
+        th.tensor([1, 2, 3]).tag("some", "diabetes", "data").send(duet, searchable=True)
+    )
 
-        guest = sy.Duet(domain_url="http://127.0.0.1:5001/")
+    guest = sy.Duet(domain_url="http://127.0.0.1:5001/")
 
-        assert len(guest.store) == 1
-        assert len(duet.store) == 2
+    assert len(guest.store) == 1
+    assert len(duet.store) == 2
 
+    try:
```

Comment:
You can use `with pytest.raises(AuthorizationException)` here instead of a `try/except`